frontend/awx/resources/inventories/inventoryHostsPage/InventoryHostFacts.tsx

```diff
@@ -0,0 +1,19 @@
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import { PageDetails } from '../../../../../framework';
+import { PageDetailCodeEditor } from '../../../../../framework/PageDetails/PageDetailCodeEditor';
+import { useGet } from '../../../../common/crud/useGet';
+import { awxAPI } from '../../../common/api/awx-utils';
+
+export function InventoryHostFacts() {
+  const { t } = useTranslation();
+  const params = useParams<{ id: string; inventory_type: string; host_id: string }>();
+
+  const { data: facts } = useGet<object>(awxAPI`/hosts/${params.host_id ?? ''}/ansible_facts`);
+
+  return (
+    <PageDetails>
+      <PageDetailCodeEditor label={t('Facts')} value={JSON.stringify(facts) || '{}'} />
```

Comment:
The old UI allows you to toggle between YAML and JSON formats here- is that something we want to carry over to the new UI as well?

Reply:
So what I've been seeing when comparing the old UI to the new UI in other pages. We don't have that toggle anymore. We are using the pageDetailCodeEditor component and I haven't seen a way to add a toggle to this component.


**Old UI**
<img width="1568" alt="Screenshot 2024-02-01 at 7 51 20 PM" src="https://github.com/ansible/ansible-ui/assets/112966709/d625dc05-faa0-4439-953b-67eef249972c">

**New UI**
<img width="1224" alt="Screenshot 2024-02-01 at 7 51 10 PM" src="https://github.com/ansible/ansible-ui/assets/112966709/d1b169f9-6976-47d8-bc72-93abdaf623b6">